ccd-definition/CaseField/CaseFieldLRspec.json

```diff
@@ -2262,13 +2262,6 @@
     "FieldTypeParameter": "ShowConditionFlags",
     "SecurityClassification": "Public"
   },
-  {
```

Comment:
Why is this field being removed?

Reply:
This field is removed because the Email Address field is added in another ticket CIV-3660 for applicant to enter the email address part.